### DISCUSSION TITLE
Better exception handling.

### DISCUSSION THREAD

**Discussion by Booklight12** [godotengine/godot-proposals#6917] [Scripting]
Add try catch key characters to make gdscript easier to use

**Comment by AThousandShips** [comment]
GDScript doesn't feature exceptions, so this would require adding them, not just adding `try` and `catch`, also this could only deal with GDScript errors, not errors from native functions and builtin classes

**Reply by AThousandShips** [reply]
Also this is a duplicate of #4736, please see there instead (that one is not suggesting it directly but is about the topic and the discussions there)

**Comment by Calinou** [comment]
- This is already being tracked in https://github.com/godotengine/godot-proposals/discussions/4736.
